src/Documentation/grains/grain_identity.md

```diff
@@ -111,10 +118,10 @@ public class ExampleGrain : Orleans.Grain, IExampleGrain
 {
     public Task Hello()
     {
-	string keyExtension;
-        long primaryKey = this.GetPrimaryKey(out keyExtension);
+		string keyExtension;
+        long primaryKey = this.GetPrimaryKeyLong(out keyExtension);
```

Comment:
I believe the spacing here is still off - but it probably looks correct on your screen. The issue appears to be that you're indenting using tabs but our convention is to use spaces.

If you're using Visual Studio Code to edit the documents (recommended), then you can switch to using spaces for indentation:
![image](https://user-images.githubusercontent.com/203839/45188953-ea826080-b279-11e8-9ebd-4ace931048e0.png)